test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -756,6 +756,36 @@ TEST(LiteInterpreterTest, GetRuntimeOpsAndInfo) {
   AT_ASSERT(runtime_ops.size() > 2900);
 }
 
+TEST(LiteInterpreterTest, isCompatibleSuccess) {
+  // test trivial success case
+  auto runtime_info = get_runtime_compatibility_info();
+  std::unordered_map<std::string, OperatorInfo> model_ops;
+  model_ops["aten::add.Scalar"] = OperatorInfo{2};
+  // arbitrarily pick bytecode version 5. This may need to be updated some time
+  // in the future
+  auto model_info = ModelCompatibilityInfo{0x5, model_ops};
+
+  AT_ASSERT(is_compatible(runtime_info, model_info));
+}
+
+TEST(LiteInterpreterTest, isCompatibleFail) {
+  // test trivial failure due to ops
+  std::unordered_map<std::string, OperatorInfo> model_ops;
+  model_ops["aten::add.Scalar"] = OperatorInfo{2};
+  // arbitrarily pick bytecode version 5. This may need to be updated some time
+  // in the future
+  auto model_info = ModelCompatibilityInfo{0x5, model_ops};
+  std::unordered_map<std::string, OperatorInfo> runtime_ops;
+  runtime_ops["aten::add.Int"] = OperatorInfo{2};
+  auto runtime_info = RuntimeCompatibilityInfo{0x6, runtime_ops};
+
+  AT_ASSERT(!is_compatible(runtime_info, model_info));
+
+  // test trivial failure due to bytecode
+  model_info.bytecode_version = 0x7;
```

Comment:
Similar above. If bytecode version is bumped to 7 this test would fail. Use `caffe2::serialize::kMaxSupportedBytecodeVersion + 1` instead.